### ISSUE TITLE
User interface: Links at bottom of Platform user interface page not working

### ISSUE THREAD

**Issue by emcdonagh-OT** [opentargets/platform#1018]
On the Platform homepage and https://www.opentargets.org/ homepage, the bottom of the page has links to:
    About
    People
    Publications
    Resources
    Data
    Jobs
    Blog
    Contact
The link for 'About', 'Data', 'Jobs' and does not work giving a 'page not found' message.

**Comment by iandunham** [comment]
Pages https://www.opentargets.org/about and https://www.opentargets.org/data no longer exist in the new structure.
Jobs should go to https://www.opentargets.org/jobs
About should be replaced by "Our Science" -> https://www.opentargets.org/science
and data should be removed  until we reinstate a data page

**Comment by deniseOme** [comment]
For consistency, I think these some of the links should reflect what is at the top of the page e.g. "People" should be "Our team".

<img width="303" alt="Screen Shot 2020-05-11 at 15 04 15" src="https://user-images.githubusercontent.com/6472381/81570809-fe5c5f00-9398-11ea-98bb-1ce0f7733ef8.png">

<img width="436" alt="Screen Shot 2020-05-11 at 15 03 14" src="https://user-images.githubusercontent.com/6472381/81570798-fbfa0500-9398-11ea-814b-822129e07c1e.png">

**Comment by andrewhercules** [comment]
I will update the link generation logic for the footer

**Comment by deniseOme** [comment]
@andrewhercules: you may also want to change the upper case X in GTEX to its lower case version as per the correct name is GTEx see the [NIH portal name](https://gtexportal.org/home/). GTEX can be found in this page:

https://www.opentargets.org/genetics-portal

Also, a suggestion re. the Jobs page. In the previous version, we use to be able to point to specific job sections e.g.

http://opentargets.org/jobs/#bioinformatian
http://opentargets.org/jobs/#statistical-geneticist

This was really handy when advertising different positions on social media.

Please note that the current address takes us to a page that says "Jobs" and one needs to scroll down to see the actual jobs. I wonder if it'd be better for usability to have the image last, as it stands in the way of seeing the list of jobs...

**Comment by andrewhercules** [comment]
I have updated the links in the footer and have also amended the labels.

@deniseOme, the jobs page is a work in progress and will be designed to showcase the benefits of joining Open Targets and where our alumni have moved to (e.g. GSK, Genomics plc, BenevolentAI, etc.) - hence the photo from last year's celebration. I checked the previous codebase and I cannot find reference to an anchor tag feature on the jobs page. As the layout of the page and underlying logic will change, it is something to consider for a later version.
